Spanner/tests/System/AdminTest.php

```diff
@@ -127,10 +129,74 @@ public function testDatabase()
         $this->assertEquals($db->ddl()[0], $stmt);
     }
 
-    public function testPgDatabase()
+    public function testCustomerManagedInstanceConfigurations()
     {
         $this->skipEmulatorTests();
         
+        $client = self::$client;
+
+        // Custom instance configuration IDs must start with 'custom' and may not contain any underscores.
+        $customConfigId = uniqid('custom-' . str_replace('_', '-', self::TESTING_PREFIX));
+
+        // Find the first instance configuration that has at least one optional replica. This indicates that it is a
+        // Google Managed multi-region config that can be used as the base config for a customer managed configuration.
+        $configurations = iterator_to_array($client->instanceConfigurations());
+        $baseConfig = array_filter($configurations, function ($configuration) {
```

Comment:
Perhaps use a `for` loop as we only need one `$baseConfig`.

Reply:
Done